src/App.tsx

```diff
@@ -32,18 +32,20 @@ function App() {
       <p className="read-the-docs">Get started by uploading two fonts</p>
       <div className="battle-ground">
         {/* Left side */}
-        <div className="side-container">
+        <div className="left-container">
```

Comment:
Are the two sides styled differently? Might need you to fill me in on the rationale here 😅